hr_employee_phone_extension/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Employee Phone Extension
+========================
+
+This module adds the following to employees :
+
+* internal_number
+* short_number
+
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
```

Comment:
Probably this comes from the template, but I find this misleading: if I follow this link I won't find further information about this module :worried: